src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form__big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label_big">
+        <input
+          type="text"
+          data-qa="keypress"
+          class="search-form__input search-form__input_big"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form__small"
+      data-qa="small"
+    >
+      <label class="search-form__label search-form__label_small">
+        <input
+          type="text"
+          data-qa="hover"
+          class="search-form__input search-form__input_small"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
   </body>
```

Comment:
HTML Formatting: Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading, but don't add them between parent and child elements.